configure.ac

```diff
@@ -7,7 +7,7 @@ AM_INIT_AUTOMAKE([foreign subdir-objects])
 
 AC_PREFIX_DEFAULT(/usr)
 
-EXTRA_TARGETS="example tests tests/unit tests/dga"
+EXTRA_TARGETS="example tests tests/dga"
```

Comment:
Is it correct to remove `tests/unit`?

Reply:
The `tests/unit` will only be added to `EXTRA_TARGETS` if json-c was found via `pkg-config` and can be successfully used for linkage. See lines 102-104.

Reply:
Thanks!